impeller/renderer/backend/vulkan/driver_info_vk.cc

```diff
@@ -6,11 +6,19 @@
 
 #include <iomanip>
 #include <sstream>
+#include <string_view>
 
 #include "flutter/fml/build_config.h"
 
 namespace impeller {
 
+/// Non functional Vulkan driver, see:
+/// https://github.com/flutter/flutter/issues/154103
+///
+/// Reports "VK_INCOMPLETE" when compiling certain entity shader with
```

Comment:
Do we know which shader this is? I'll try to follow up.

We can catch up afterwards about this. Not blocking.

Reply:
Porterduff and conical gradient ssbo